src/eyreish/wrapper.rs

```diff
@@ -232,3 +232,88 @@ impl<C> StdError for WithSourceCode<Report, C> {
         self.error.source()
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use thiserror::Error;
+
+    use crate::{Diagnostic, LabeledSpan, Report, SourceCode, SourceSpan};
+
+    #[derive(Error, Debug)]
```

Comment:
I couldn't use derive macro here. Looks like it always expands to `miette::`, instead of `crate::`. This may be fixed with `$crate::` in macro implementation, I think

Reply:
This would be a good patch to have